net/frr/src/opnsense/mvc/app/controllers/OPNsense/Quagga/forms/dialogEditOSPFNetwork.xml

```diff
@@ -20,6 +20,12 @@
     <type>text</type>
     <help>Area in wildcard mask style like 0.0.0.0 and no decimal 0</help>
   </field>
+  <field>
+    <id>network.arearange</id>
+    <label>Area Range</label>
+    <type>text</type>
+    <help>To a network to summarize for this area (e.g. 192.168.0.0/23).</help>
```

Comment:
the help string needs improvement.

Reply:
Indeed, I flipped the sentence twice and forgot to double check. :)